.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "trailingComma": "none",
+  "semi": false
+}
```

Comment:
Do we not already have prettier setup for backend?

Reply:
this is to override someone's local prettier changes. If this file will be there, people can use prettier locally in the vs code.

Reply:
Need to confirm with @ankurnarkhede if we need this

Reply:
Removed this prettier settings file